qiita_pet/templates/private_studies.html

```diff
@@ -15,7 +15,7 @@
         });
         $('#shared-studies-table').dataTable({
             order: [[ 0, "asc" ]],
-            columnDefs: [{type:'natural', targets:[1,2,3,7]}]
+            columnDefs: [{type:'natural', targets:[2,3,4]}]
```

Comment:
It's just all the columns that don't have HTML in them. The HTML throws off the natsort plugin.